### DISCUSSION TITLE
[feat] How can multi-user login credentials be isolated

### DISCUSSION THREAD

**Discussion by xuyongweb** [tauri-apps/tauri#8002] [Q&A]
### Describe the problem

How to isolate user credentials for multi-user login?
At present, localstorage is used to store tokens to determine whether to log in, but the credentials of the later login will overwrite the credentials of the first login. The technical station used is vue3 + vuex, and the login window jumps to the main window and the vuex content disappears

### Describe the solution you'd like

The localstorage content of different login users is isolated from each other

**Comment by algesthesiah** [comment]
<img width="801" alt="image" src="https://github.com/tauri-apps/tauri/assets/26083451/2eab5831-43fa-43e3-bc65-df2543bb7051">
The official Tauri multi-window example seems to indicate that multiple windows share the same local storage.
